src/lib.rs

```diff
@@ -65,215 +98,2254 @@ pub struct Leser {
+#[xml(root = b"DerogationRemedialActionType")]
+pub struct DerogationRemedialActionType {
+    #[xml(
+        ns = b"xwas",
+        name = b"derogationRemedialActionIdentifier",
+        ty = "child"
+    )]
+    pub derogation_remedial_action_identifier: String, // xs:ID
+    #[xml(ns = b"xwas", name = b"derogationRemedialAction", ty = "child")]
+    pub derogation_remedial_action: CodeAbhilfemassnahmeType,
+    #[xml(
+        ns = b"xwas",
+        name = b"derogationRemedialActionStartDate",
+        ty = "child"
+    )]
+    pub derogation_remedial_action_start_date: String,
+    #[xml(ns = b"xwas", name = b"derogationRemedialActionEndDate", ty = "child")]
+    pub derogation_remedial_action_end_date: String,
+    #[xml(ns = b"xwas", name = b"derogationRemedialActionCost", ty = "child")]
+    pub derogation_remedial_action_cost: f64,
+    #[xml(ns = b"xwas", name = b"remarks", ty = "child")]
+    pub remarks: Option<String>,
+    #[xml(ns = b"xwas", name = b"_id", ty = "attr")]
     #[serde(skip)]
-    _xmlns_xsi: ConstStr,
+    pub _id: ConstStr,
+}
+
+#[derive(Default, Debug, XmlSerialize, XmlDeserialize, Serialize, Deserialize, Tsify)]
+#[xml(root = b"codeShapthParameterType")]
+pub struct CodeShapthParameterType {
+    #[xml(ns = b"xwas", name = b"code", ty = "child")]
+    pub code: String,
+    #[xml(ns = b"xwas", name = b"name", ty = "child")]
+    pub name: Option<String>,
+}
+
+#[derive(Default, Debug, XmlSerialize, XmlDeserialize, Serialize, Deserialize, Tsify)]
+#[xml(root = b"codeProbennahmezeitraumType")]
+pub struct CodeProbennahmezeitraumType {
+    #[xml(ns = b"xwas", name = b"code", ty = "child")]
+    pub code: String,
+    #[xml(ns = b"xwas", name = b"name", ty = "child")]
+    pub name: Option<String>,
+}
+
+#[derive(Default, Debug, XmlSerialize, XmlDeserialize, Serialize, Deserialize, Tsify)]
+#[xml(root = b"codeArtProbennahmestelleEUType")]
+pub struct CodeArtProbennahmestelleEUType {
+    #[xml(ns = b"xwas", name = b"code", ty = "child")]
+    pub code: String,
+    #[xml(ns = b"xwas", name = b"name", ty = "child")]
+    pub name: Option<String>,
+}
+
+/// Transport der Informationen, die für Qualität und Überwachung gem. dem neuen Vorgaben
+/// für das EU-Berichtsformat benötigt werden.
+#[derive(Default, Debug, XmlSerialize, XmlDeserialize, Serialize, Deserialize, Tsify)]
+#[xml(root = b"QualityAndMonitoringType")]
+pub struct QualityAndMonitoringType {
     #[xml(
-        ns = b"xsi",
-        name = b"schemaLocation",
-        ty = "attr",
-        value = "xwasser xwasser.xsd"
+        ns = b"xwas",
+        name = b"qualityAndMonitoringRequirementIdentifier",
+        ty = "child"
     )]
+    pub quality_and_monitoring_requirement_identifier: String, // bn-uq-g2g:UUID
+    #[xml(ns = b"xwas", name = b"parameterCode", ty = "child")]
+    pub parameter_code: CodeShapthParameterType,
+    #[xml(ns = b"xwas", name = b"parameterThresholdValue", ty = "child")]
+    pub parameter_threshold_value: f64,
```

Comment:
With this I have some doubts, because we do not use these values as numbers, but our users would not expect to see different number due to float number usage.